src/node/tls.ts

```diff
@@ -23,9 +23,13 @@
 // OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE
 // USE OR OTHER DEALINGS IN THE SOFTWARE.
 
+import { checkServerIdentity } from 'node-internal:internal_tls';
```

Comment:
should `SecureContext` be export too?

If not `SecureContext` will come from the polyfill and `createSecureContext() instanceof SecureContext` will be false

Reply:
Yes you're right.